### PR TITLE
Docblock tribe_is_day function and filter

### PR DESCRIPTION
**Pull request by deblynprado**

### CODE REVIEW

File: src/functions/template-tags/day.php [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
 	 */
 	function tribe_is_day(): bool {
 		$context  = tribe_context();
 		$is_day = tec_is_view( \Tribe\Events\Views\V2\Views\Day_View::get_view_slug() );
 
 		/**
-		 * Allows filtering of the tribe_is_day boolean value.
+		 * Filters the boolean value indicating if the current view is the Day View.
 		 *
-		 * @since 6.0.7 Added context to parameters.
```
Comment (bordoni) [inline src/functions/template-tags/day.php:17]: Why did we remove the `@since`?

**Review by bordoni — CHANGES_REQUESTED** [review]

**Comment by bordoni** [comment]
PR is currently missing labels

File: src/functions/template-tags/day.php (hunk 1 above)
Comment (codingmusician) [reply src/functions/template-tags/day.php:17]: `@since` should have been moved below the description and above the `@param` list.

Comment (deblynprado) [reply src/functions/template-tags/day.php:17]: I should move the `@since` but removed instead, my bad.

**Comment by bordoni** [comment]
Move to `release/T24.babayaga`

**Pull request closed without merging**